QUEST_LV_0200.tsv

```diff
@@ -3317,33 +3317,33 @@ QUEST_LV_0200_20150323_003316	$When I thought about the moment when this scriptu
 QUEST_LV_0200_20150323_003319	Who are you? Become the slave of the Succubus!
-QUEST_LV_0200_20150323_003320	I can't say that this land is full of milk and honey anymore.{nl}Should I give up on bee raising?
-QUEST_LV_0200_20150323_003321	The bees turned into violent monsters.{nl}I don't know how to live on from now.
-QUEST_LV_0200_20150323_003322	I don't remember what happened after I was casted with the curse.{nl}But, those demons, they must be definitely looking for something.{nl}
+QUEST_LV_0200_20150323_003320	$I can't say milk and honey flows in this land anymore.{nl}Should I give up on bee farming?
+QUEST_LV_0200_20150323_003321	$The bees have turned into violent monsters.{nl}I don't know how I can make a living from now on.
+QUEST_LV_0200_20150323_003322	I don't remember what happened after I was casted with the curse.{nl}But, those demons, they definitely must be looking for something.{nl}
 QUEST_LV_0200_20150323_003323	$If you are going to the Great Cathedral, I want to stop you.{nl}The demon who casted the curse on this land went to the Great Cathedral.
 QUEST_LV_0200_20150323_003324	$I saw the true nature of the curse with my own eyes.{nl}That Succubus beside Naktis.. Even I, who was silently watching, almost got seduced.
-QUEST_LV_0200_20150323_003325	Even on Medis Diena, this place was safe.{nl}I used to think a disaster is someone else's story.
-QUEST_LV_0200_20150323_003326	The bees are not the only ones that changed.{nl}The monsters that I saw for the first time in my life were coming over to Siaulai Forest continuously.
-QUEST_LV_0200_20150323_003327	I spent many years to make the honey alcohol, but I have to end now without any siginificant result.{nl}Who should I blame for this..
+QUEST_LV_0200_20150323_003325	$Even during Medis Diena, this place was safe.{nl}I used to think disasters were somebody else's problem.
+QUEST_LV_0200_20150323_003326	$The bees are not the only ones that changed.{nl}The monsters that I saw for the first time in my life had overrun Siauliai Forest back then.
+QUEST_LV_0200_20150323_003327	$I've spent many years making mead, but I have to end now without any significant product.{nl}Who should I blame for this..
 QUEST_LV_0200_20150323_003328	{memo X}All other Revelators were wrong.{nl}But if it were you.. I don't know.
-QUEST_LV_0200_20150323_003329	The priests told us to wait, and the damages that were caused by the monsters were bad.{nl}I could not stand anymore so I stood up myself.
-QUEST_LV_0200_20150323_003330	I will find out why monsters are making mess here.{nl}Then the priests will not say anything.
+QUEST_LV_0200_20150323_003329	The priest told us to wait, but the damage caused by the monsters was getting bad.{nl}I could not stand it anymore so I took action.
+QUEST_LV_0200_20150323_003330	I will find out why the monsters are making a mess here.{nl}Then the priest will be speechless.
```

Comment:
Priest -> priests 

(Cant check the korean now, but if I remember correctly, it said 'priests' for these quests)